source/fe/mapping_q_generic.cc

```diff
@@ -764,32 +764,30 @@ initialize_face (const UpdateFlags      update_flags,
         {
           aux.resize (dim-1, std::vector<Tensor<1,spacedim> > (n_original_q_points));
 
-          // Compute tangentials to the
-          // unit cell.
-          const unsigned int nfaces = GeometryInfo<dim>::faces_per_cell;
-          unit_tangentials.resize (nfaces*(dim-1),
-                                   std::vector<Tensor<1,dim> > (n_original_q_points));
+          // Compute tangentials to the unit cell.
+          for (unsigned int i=0; i!=GeometryInfo<dim>::faces_per_cell * (dim-1); ++i)
```

Comment:
While they are equivalent, I think `unit_tangentials.size()` would be a little nicer here.